system-helper/org.freedesktop.Flatpak.policy.in

```diff
@@ -127,6 +127,17 @@
     </defaults>
   </action>
 
+  <action id="org.freedesktop.Flatpak.update-remote">
+    <description>Update Remote</description>
+    <message>Authentication is required to update the configuration of software repositories</message>
+    <icon_name>package-x-generic</icon_name>
+    <defaults>
+      <allow_any>auth_admin</allow_any>
```

Comment:
I think we need to change this to yes in order to allow non-admin to update use the action, right?